ts/DataGrid/DataGrid.ts

```diff
@@ -514,6 +520,10 @@ class DataGrid {
         this.container.addEventListener('mouseover', (e): void => {
             this.handleMouseOver(e);
         });
+
+        new ResizeObserver((): void => {
```

Comment:
Should we add this as a class prop, so DataGridComponent can call `observer.disconnect` on `destroy()`? (Possibly we should also have destroy on the DataGrid, but that's a larger matter)